src/looper.ts

```diff
@@ -121,7 +134,10 @@ function loopProbe(
     if (counter === repeats) {
       clearInterval(probeInterval)
     } else {
-      doProbe(++counter, probe, notifications)
+      // eslint-disable-next-line no-lonely-if
```

Comment:
How about using else if instead?

Reply:
i think `else if` is better used to compare the same variable. In this case `if (counter === repeats)` and `if (isConnectedToSTUNServer)` doesn't compare same variable, so not a good case to use `else if`. What do you think?

Reply:
i think @gsukanto is correct, can just use else if here because there are no other conditionals inside the else block.